src/app/organizations/organization-details/organization-details.component.ts

```diff
@@ -126,4 +127,25 @@ export class OrganizationDetailsComponent implements OnInit {
                 displayBackendError(err);
             }, () => SpinnerUtil.hideSpinner());
     }
+
+    deleteMember(orgID: string, memberID: string) {
+        SpinnerUtil.showSpinner();
+        this.organizationService.removeMemberFromOrganization(orgID, memberID)
+            .subscribe(() => {
+                    SpinnerUtil.hideSpinner();
+                    swal({
+                        title: '',
+                        text: 'Success!',
+                        type: 'success'
+                    }).then(function () {
+                        this.reloadPage('/dash/manage_groups/' + this.organization.uuid);
```

Comment:
Use string interpolation (<code>`/dash/manage_groups/${this.organization.uuid}`</code>).
As we talked previously, you don't need to reload the entire page. Rather refresh users with `getMembersForOrganization()` method.

Reply:
OK